components/com_content/src/Model/ArticlesModel.php

```diff
@@ -572,7 +572,7 @@ protected function getListQuery()
 				$relativeDate = (int) $this->getState('filter.relative_date', 0);
 				$query->where(
 					$db->quoteName($dateField) . ' IS NOT NULL AND '
-					. $db->quoteName($dateField) . ' >= ' . $query->dateAdd($nowDate, -1 * $relativeDate, 'DAY')
+					. $db->quoteName($dateField) . ' >= ' . $query->dateAdd($db->quote($nowDate), -1 * $relativeDate, 'DAY')
```

Comment:
Actually it should be a prepared parameter
```suggestion
					. $db->quoteName($dateField) . ' >= ' . $query->dateAdd(':relativeNowDate ', -1 * $relativeDate, 'DAY')
					->bind(':relativeNowDate ', $nowDate)
```